src/main/java/org/zone/commands/structure/misc/Messages.java

```diff
@@ -0,0 +1,304 @@
+    }
+
+    public static Component getUniversalFlagMessageView(Component message) {
+        return getUniversalMessageTag().append(message).color(NamedTextColor.AQUA);
+    }
+
+    public static Component getUniversalUpdatedFlag(@SuppressWarnings("TypeMayBeWeakened") FlagType<?> type) {
+        return Component.text("Updated " + type.getName()).color(NamedTextColor.AQUA);
+    }
+    //Universal Messages end
+    //Universal only for some special classes
+
+    //Universal only for some special classes end or will continue
+
+    //Custom messages
+    public static Component getLoadingZonesStart() {
+        return Component.text("|---|Loading Zones|---|").color(NamedTextColor.AQUA);
+    }
+
+    public static Component getZonesLoadingfrom(String zonesFolderpath) {
+        return Component.text("|- Loading from '" + zonesFolderpath + "'").color(NamedTextColor.DARK_AQUA);
+    }
+
+    public static Component getZonesLoadingFail(String keyfilepath) {
+        return Component
+                .text("Could not load zone of '" +
+                              keyfilepath +
+                              "'. Below is details on why (this is not a crash)")
+                .color(NamedTextColor.RED);
+    }
+
+    public static Component getZonesLoaded(Collection<Zone> zoneCollection) {
+        return Component
+                .text("|---|Loaded " + zoneCollection.size() + " Zones|---|")
+                .color(NamedTextColor.AQUA);
+    }
+
+    public static Component getZoneSpongeCommandnotEmptyErrorElse() {
+        return Component.text("Unknown error").color(NamedTextColor.RED);
+    }
+
+    public static Component getZoneSpongeCommandnotgetHasPermission(ArgumentCommand argumentCommand) {
+        return Component
+                .text(" You do not have permission for that command" +
+                              ". You" +
+                              " require " +
+                              argumentCommand.getPermissionNode())
+                .color(NamedTextColor.RED);
+    }
+
+    public static Component getZonesCreateEndCommandrunopZoneEmptyError() {
+        return Component.text("A region needs to be started. Use /zone create bounds " + "<name...>")
+                .color(NamedTextColor.RED);
+    }
+
+    public static Component getZonesCreateEndCommandrunopParentEmptyError(Zone zone) {
+        return Component.text("Could not find parent zone of " + zone.getParentId().orElse("None")).color(NamedTextColor.RED);
+    }
+
+    public static Component getZonesCreateEndCommandrunError2(@SuppressWarnings("TypeMayBeWeakened") Zone parent) {
+        return Component.text("Region must be within " + parent.getId()).color(NamedTextColor.RED);
+    }
+
+    public static Component getZonesCreateEndCommandrunError3(@SuppressWarnings("TypeMayBeWeakened") Zone parent) {
+        return Component.text("Region must be within " + parent.getId()).color(NamedTextColor.RED);
```

Comment:
This can be combined with the method above

Reply:
which

Reply:
ohk